src/panels/lovelace/editor/card-editor/hui-edit-card.ts

```diff
@@ -428,7 +428,7 @@ export class HuiEditCard extends LitElement {
         }
 
         @media (min-width: 1200px) {
-          paper-dialog {
+          ha-paper-dialog {
```

Comment:
This seems weird. If there is no paper dialog on this page, we can remove this CSS Declaration ?

Reply:
But there is. https://github.com/home-assistant/home-assistant-polymer/blob/a9bd53432e84a05a4933a09470468f4e4efcd6fc/src/panels/lovelace/editor/card-editor/hui-edit-card.ts#L151

Reply:
Oh it was broken before 😛